Sakura/state.py

```
@@ -15,5 +15,4 @@
 valkey_client: Optional[AsyncValkey] = None
 payment_storage: Dict[str, dict] = {}
 effects_client: Optional[TelegramClient] = None
-gemini_client = None
-openrouter_client = None
\ No newline at end of file
+openrouter_clients: List[AsyncOpenAI] = []
```

Ruff (0.13.3)

Undefined name `AsyncOpenAI`

(F821)